code/api/src/modules/user/resolvers.js

```diff
@@ -68,6 +68,26 @@ export async function getAll() {
   return await models.User.findAll()
 }
 
+// Update user
+export async function update(parentValue, { id, name, email, password, image, address, description }, { auth }) {
+  if(auth.user) {
+    var updatedUser = await models.User.update(
+      {
+        name,
+        email,
+        password,
+        image,
+        address,
+        description
+      },
+      {where: {id}, returning: true}
+    )
+    return updatedUser[1][0].dataValues;
+  } else {
```

Comment:
Resourceful way here to parse through the `updatedUser` response to access the data - looks good!